evm/its.js

```diff
@@ -425,51 +418,52 @@ async function processCommand(config, chain, options) {
             break;
         }
 
-        case 'setTrustedAddress': {
+        case 'set-trusted-address': {
+            const [trustedChain, trustedAddress] = args;
             const owner = await new Contract(interchainTokenService.address, IOwnable.abi, wallet).owner();
 
             if (owner.toLowerCase() !== walletAddress.toLowerCase()) {
                 throw new Error(`${action} can only be performed by contract owner: ${owner}`);
             }
 
-            validateParameters({ isNonEmptyString: { trustedChain: options.trustedChain } });
+            validateParameters({ isNonEmptyString: { trustedChain } });
 
             let trustedChains, trustedAddresses;
 
-            if (options.trustedChain === 'all') {
+            if (trustedChain === 'all') {
                 const itsChains = Object.values(config.chains).filter((chain) => chain.contracts?.InterchainTokenService?.skip !== true);
                 trustedChains = itsChains.map((chain) => chain.axelarId);
-                trustedAddresses = itsChains.map((_) => options.trustedAddress || chain.contracts?.InterchainTokenService?.address);
+
+                trustedAddresses = trustedChains.map((_) => trustedAddress || chain.contracts?.InterchainTokenService?.address);
             } else {
-                const trustedChain =
-                    getChainConfig(config, options.trustedChain.toLowerCase(), { skipCheck: true })?.axelarId ||
-                    options.trustedChain.toLowerCase();
-                const trustedAddress =
-                    options.trustedAddress ||
-                    getChainConfig(config, options.trustedChain.toLowerCase())?.contracts?.InterchainTokenService?.address;
-
-                if (trustedChain === undefined || trustedAddress === undefined) {
-                    throw new Error(`Invalid chain/address: ${options.trustedChain}`);
+                const trustedChainFinal =
```

Comment:
use better variable names, this reminds me of university assignment solutions :p